test/unit/formidable.test.js

```diff
@@ -159,6 +160,92 @@ function makeHeader(filename) {
     form.emit('end');
   });
 
```

Comment:
More flat `test` with more "longer" titles are preferable. And use `allowEmptyFiles: false` for example.

Reply:
In the way, you are suggesting I will repeat the text(e.g: `when not allow empty files'`) across multiple the describe and I thin they separated in multiple describes they are logical grouped. But I can change if it doesn't fit the test describe patterns used in this repo.

Reply:
it is ok we can change it a future commit